runtime/doc/options.txt

```diff
@@ -2384,16 +2384,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 	  item		default		Used for ~
 	  stl:c		' ' or '^'	statusline of the current window
 	  stlnc:c	' ' or '='	statusline of the non-current windows
-	  vert:c	'|'		vertical separators |:vsplit|
-	  fold:c	'-'		filling 'foldtext'
+	  vert:c	'│'		vertical separators |:vsplit|
+	  fold:c	'·'		filling 'foldtext'
 	  diff:c	'-'		deleted lines of the 'diff' option
 
 	Any one that is omitted will fall back to the default.  For "stl" and
```

Comment:
This doesn't work for some reason, even with the old defaults.

Reply:
yep that 's something that is a bit strange as the defualt is returned by a function specific to the fillchar in question. I would favor defining a default right into the structure as in https://github.com/neovim/neovim/pull/6073/files#diff-bfacc6d8f849b9559c1c6cd5b79a5431R3367